exchange/exchange.go

```diff
@@ -1330,12 +1330,6 @@ func makeBidExtJSON(ext json.RawMessage, prebid *openrtb_ext.ExtBidPrebid, impEx
 		prebid.Meta = &metaContainer.Prebid.Meta
 	}
 
-	if prebid.Meta == nil {
-		prebid.Meta = &openrtb_ext.ExtBidPrebidMeta{}
-	}
-
-	prebid.Meta.AdapterCode = adapter.String()
```

Comment:
Why is it necessary to move this code out this function? Looks like the functional change is adapters can populate the AdapterCode and take priority.